.github/workflows/ci.yml

```diff
@@ -398,7 +398,7 @@ jobs:
           ndk: 21.0.6113669
           arch: x86
           cmake: 3.10.2.4988404
-          script: flutter build apk --debug --target=test_driver/app.dart && flutter install && flutter drive --target=test_driver/app.dart --dart-define=testName="" --suppress-analytics
+          script: flutter build apk --debug --target=test_driver/app.dart && flutter install --debug && flutter drive --target=test_driver/app.dart --dart-define=testName="" --suppress-analytics
```

Comment:
The default profile for `flutter build apk` and `flutter install` is changed to `release` for flutter beta version. We have to explicitly set them both to `debug`, since `flutter drive` requires `debug` for non-web. 
Here is the result of the command in `release`:

`flutter drive --release --target=test_driver/app.dart --dart-define=testName="" --suppress-analytics`

```
Flutter Driver (non-web) does not support running in release mode.

Use --profile mode for testing application performance.
Use --debug (default) mode for testing correctness (with assertions).
```